src/System.Diagnostics.Process/tests/ProcessTests.cs

```diff
@@ -575,7 +575,23 @@ public static IEnumerable<object[]> GetTestProcess()
             yield return new object[] { currentProcess, Process.GetProcessesByName(currentProcess.ProcessName, "127.0.0.1").Where(p => p.Id == currentProcess.Id).Single() };
         }
 
-        [Theory, PlatformSpecific(PlatformID.Windows)]
+        private static bool ProcessPeformanceCounterEnabled()
+        {
+            try
+            {
+                int? value = (int?)Microsoft.Win32.Registry.GetValue(@"HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Services\PerfProc\Performance", "Disable Performance Counters", null);
```

Comment:
I see.  No, it shouldn't be invoked in that case.

So I guess as long as it compiles, it'll be ok.  Will it compile?  Or does something special need to be done to pull in the "not supported" Unix build of Microsoft.Win32.Registry?